volume/volume.go

```diff
@@ -11,7 +11,11 @@ import (
 
 // DefaultDriverName is the driver name used for the driver
```

Comment:
This comment should be update (or removed and one comment on each const) :angel: